### PR TITLE
Feature/player 2947

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Add UI component if multiAudio is available.
Unit tests for functions added, for UI will be added after UI implemented. 
Unit tests were run without errors

### CODE REVIEW

File: js/components/controlBar.js [hunk 1]
```diff
@@ -753,6 +797,11 @@ var ControlBar = React.createClass({
         continue;
       }
 
+      if (defaultItems[k].name === "multiAudio" &&
```
Comment (IvanSakharovskii) [inline js/components/controlBar.js]: What `k` it is? Rename please

File: js/components/multi-audio/multiAudioPopover.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+var React = require('react'),
+  CONSTANTS = require('../../constants/constants'),
+  AccessibleMenu = require('../higher-order/accessibleMenu'),
+  CloseButton = require('../closeButton');
+
+var MultiAudioPopover = React.createClass({
+  handleClose: function() {
```
Comment (IvanSakharovskii) [inline js/components/multi-audio/multiAudioPopover.js]: jsdoc

File: js/controller.js [hunk 3]
```diff
@@ -859,7 +867,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
      * @param multiAudio.multiAudio {Array} - list of objects with data for each audio
      */
     onMultiAudioFetched: function(event, multiAudio) {
-    //  TODO: code for a video with multiaudio should be here
+      this.state.multiAudio = multiAudio;
+    },
+
+    setCurrentAudio: function(id) {
```
Comment (IvanSakharovskii) [inline js/controller.js]: jsDoc

File: js/controller.js [hunk 4]
```diff
@@ -859,7 +867,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
...
+      this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, id);
+      this.setCurrentAudioId(id);
+    },
+
+    setCurrentAudioId: function(id) {
```
Comment (IvanSakharovskii) [inline js/controller.js]: jsDoc

File: js/components/multi-audio/multiAudioPopover.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply js/components/multi-audio/multiAudioPopover.js]: \+

File: js/controller.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply js/controller.js]: \+

File: js/controller.js (hunk 4 above)
Comment (KseniiaSokolovaEpam) [reply js/controller.js]: \+

File: js/components/controlBar.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/components/controlBar.js]: I think it would be better to create new task to rename old variables.

**Pull request merged**